transpiler/src/zero_generator.rs

```diff
@@ -11,9 +8,8 @@ use zero_machine_code::instructions::*;
 //Struct that enables transpilation management into System Zero instructions by keeping track of the stack frames.
 struct Transpiler {
     instructions: Vec<GeneralInstruction>,
-    current_stack_frame: StackFrame,
-    previous_stack_frames: Vec<StackFrame>,
-    label_count: usize,
+    stack_frame: StackFrame,
+    if_label_count: usize,
```

Comment:
couldn't other types of transpilations, e.g. for loops, require the generation of labels?

Reply:
Yes! I was thinking those would be separate counters though? And we'd have other functions to generate those labels (say, as "for{number}")

Reply:
I think we could give them different namings, but keep a single counter since it doesn't matter if we skip numberings for things, all that matters is them never repeating